performance_manager/lib/logging_utils.py

```diff
@@ -18,70 +17,75 @@ def __init__(self, process_name: str, **metadata: MdValues) -> None:
         create a process logger with a name and optional metadata. a start time
         and uuid will be created for timing and unique identification
         """
-        self.parent = "performance_manager"
-        self.process_name = process_name
-        self.metadata = metadata
-        self.uuid: Optional[uuid.UUID] = None
+        self.default_data: Dict[str, Any] = {}
+        self.metadata: Dict[str, Any] = {}
```

Comment:
store logging data as two dictionary objects, one for `default_data` (parent, process_name, uuid, duration, error_type, etc...) another for any `metadata`.